### ISSUE TITLE
Bug: Can't Enable autostart again

### ISSUE THREAD

**Issue by JesterHeartes** [Friends-of-Monika/mas-autostart#3]
### What happened?

I installed the mod and after the event with Monika talking about it i let her disable the autostart function to test it. When i wanted to let her enable it again, she isn't able to.

### Log or traceback output

```shell
[2022-07-09 19:17:45] [!ERROR! T_T]: [MAS Autostart Mod] Got non-zero exit code from shortcut script invocation (1.)

this is from the submod_log.log file after Monika tried Enable the autostart.
```

### Submod version

1.1.1

### Monika After Story version

v0.12.8 (the latest one as far as i know)

### Unstable

No

**Comment by dreamscached** [comment]
Could you type `winver.exe` in <kbd>Win+R</kbd> and tell us windows version and build?

Edit: could reproduce on my machine, debugging. Thanks for your feedback!

**Comment by JesterHeartes** [comment]
No problem, happy if it can help :)

**Comment by dreamscached** [comment]
Fixed in release 1.1.2, be sure to update the submod either via Submod Updater Plugin or manually. Thanks for reporting!
